src/index.js

```diff
@@ -12,83 +12,34 @@ document.addEventListener('DOMContentLoaded', () => {
     }
     const roomButton = renderElement(roomList, 'button', ['row', 'btn', 'btn-light'], 'Add Room');
     roomList.setAttribute('data-user-id', res[0].id)
-    roomButton.addEventListener('click', function(){
-      renderNewRoomText(roomList, roomButton);
+    roomButton.addEventListener('click', function() {
+      renderEntityText(roomEndPoint, "room", roomList, roomButton);
```

Comment:
I don't think we need to pass roomEndpoint here since this is a global variable, we still can access it inside the function